pkg/run/random_ports.go

```diff
@@ -0,0 +1,30 @@
+package run
+
+import "net"
+
+// GetRandomPorts returns a slice of random ports.
+func GetRandomPorts(count int) (ports []int32, retErr error) {
```

Comment:
This doesn't just get random port but available/unused ports, right? Would something like `GetUnusedPorts` be a better name?

Reply:
ack